src/pages/ItemsList/itemList.scss

```
@@ -0,0 +1,310 @@
+@import '../../styles/variables.scss';
+@import './variablesForItemList.scss';
+@import url('https://fonts.googleapis.com/css2?family=Gothic+A1&display=swap');
+.itemList {
```

```suggestion
@import url('https://fonts.googleapis.com/css2?family=Gothic+A1&display=swap');

.itemList {
```
이런 부분은 가독성을 위해 한칸씩 띄어서 작성해주세요!

---

완료